packages/@aws-cdk/aws-applicationautoscaling/lib/target-tracking-scaling-policy.ts

```diff
@@ -35,14 +35,24 @@ export interface BaseTargetTrackingProps {
   /**
    * Period after a scale in activity completes before another scale in activity can start.
    *
-   * @default - No scale in cooldown.
+   * @default Duration.seconds(300) for the following scalable targets: ECS services,
+   * Spot Fleet requests, EMR clusters, AppStream 2.0 fleets, Aurora DB clusters,
```

Comment:
If I'm understanding correctly this is the default AWS behavior and not a default value set by the CDK, should we use `Duration.seconds` in the description for values not set by the CDK?

Reply:
Yes because this property is of type `cdk.Duration`